File: Python/URI/Beginner/1036.py
import sys
from math import *

a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

delta = pow(b,2) - (4*a*c)

if (a == 0) or (delta < 0):
	print ("Impossivel calcular")
else:
	x1 = (-b + sqrt(delta)) / (2*a) 
	x2 = (-b - sqrt(delta)) / (2*a)
	print ("R1 = %.5f" % x1)
	print ("R2 = %.5f" % x2)

File: URI/Beginner/1036.cpp
#include <iostream>
#include <cstdio>
#include <cstdlib>
#include <string>
#include <math.h>

using namespace std;

int main()
{
    double a, b, c;
    cin >> a >> b >> c;

    double delta = pow(b, 2) - (4 * a * c);

    if (delta < 0 || delta == 0 || a == 0.0)
    {
        cout << "Impossivel calcular" << endl;
    }
    else
    {
        double raiz1 = ((b * (-1)) + sqrt(delta)) / (2 * a);
        double raiz2 = ((b * (-1)) - sqrt(delta)) / (2 * a);

        printf("R1 = %.5lf\n", raiz1);
        printf("R2 = %.5lf\n", raiz2);
    }

    return 0;
}